### ISSUE TITLE
Details List: Group header doesn't announce number of children

### ISSUE THREAD

**Issue by bcoard** [OfficeDev/office-ui-fabric-react#11818]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:

When a Narrator user navigates to a group header, the item doesn't announce how many children it has.

### Please provide a reproduction of the issue in a codepen:

1. Navigate to [this](https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/grouped) page.
2. Turn Narrator on.
3. Using keyboard navigation go to the group header.
